de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc

```diff
@@ -1193,9 +1193,70 @@ Now, in the _Navigator_ double click on ConfigurationTree and scroll to the Mass
 Here you can find a budget breakdown.
 |image:chapterModeling/MassConceptBudget.png[Insepcting the mass budegt breakdown]
 
+|===
+==== Functional Electrical Architecture
+===== Add Interface to the model
+Virtual Satellite enables the user to add _Interfaces_ to the system model. Add the concept *Functional Electrical Architecture* as described in section in <<Activating a Concept>>
+Elements of the functional electrical architecture can be added to the system model by opening the context menu with the right mouse and click. 
+
+There are _InterfaceEnds_ that are of a specific _InterfaceType_  and have the potential to connect _Interfaces_.
+
+_InterfaceTypes_  need to be defined as child elements of an _InterfaceTypeCollection_ .
+The _InterfaceTypeCollection_  is created by clicking with the right mouse on the system model and select menu:Context[Functional Electrical Architecture>Add InterfaceTypeCollection].
+image:chapterModeling/subsectionFEA/InterfaceType.png[InterfaceType_Collection]
+A type can be created by clicking with the right mouse on the _InterfaceTypeCollection_ and selecting menu:Context[Functional>Electrical Architecture>Add InterfaceType]
+image:chapterModeling/subsectionFEA/AddInterfaceType.png[InterfaceType]
+
+An _InterfaceType_ can be a _DataInterfaceType_ , a _ThermalReferencePoint_ or a _PowerInterfaceType_.
+
+By adding a _RangedVoltageDefinition_ or a _FixedVoltageDefinition_ to the _PowerInterfaceType_ a minimum and maximum voltage  can be specified.
+
+image:chapterModeling/subsectionFEA/VoltageSpecificationPowerInterfaceType.png[Specify the voltage of a power Interface]
+
+_InterfaceEnds_  should be defined in a product tree as this definition enables elements of the configuration tree to inherit the properties defined inside the product tree. It is added to the product tree by clicking with the right mouse on the menu:Context[product tree >Functional Electrical Architecture>Add _Interface_End].

```

Comment:
We should try to have 1 sentence per line.

Reply:
Done